subnet/subnet.go

```diff
@@ -96,5 +105,7 @@ type Manager interface {
 	GetNetworkConfig(ctx context.Context, network string) (*Config, error)
 	AcquireLease(ctx context.Context, network string, attrs *LeaseAttrs) (*Lease, error)
 	RenewLease(ctx context.Context, network string, lease *Lease) error
-	WatchLeases(ctx context.Context, network string, cursor interface{}) (WatchResult, error)
+	WatchLeases(ctx context.Context, network string, cursor interface{}) (LeaseWatchResult, error)
+	GetNetworks(ctx context.Context) ([]string, error)
```

Comment:
Let's combine `GetNetworks` and `WatchNetworks` into one. `GetNetworks` is just a watch with `nil` cursor. I'm actually working on another feature and doing exactly that for leases. We can rename `WatchNetworks` to `GetNetworks` if you think it's better.